gctscript/vm/vm_types.go

```diff
@@ -0,0 +1,45 @@
+package vm
+
+import (
+	"context"
+	"sync"
+	"time"
+
+	"github.com/d5/tengo/script"
+	"github.com/gofrs/uuid"
+)
+
+const (
+	// AuditEventName name to use for audit event logging
+	AuditEventName = "gctscript"
+
+	// DefaultTimeoutValue default timeout value for virtual machines
+	DefaultTimeoutValue = 1 * time.Minute
+)
+
+var (
+	// pool stuff
```

Comment:
probably change comment

Reply:
but it is pool stuff the comment doesn't lie it just isn't helpful!
will change it :D